staging/src/k8s.io/kube-gen/cmd/client-gen/generators/fake/generator_fake_for_group.go

```diff
@@ -70,18 +71,20 @@ func (g *genFakeForGroup) GenerateType(c *generator.Context, t *types.Type, w io
 
 	sw.Do(groupClientTemplate, m)
 	for _, t := range g.types {
+		tags, err := util.ParseClientGenTags(t.SecondClosestCommentLines)
```

Comment:
I don't really like that we concentrate on SecondClosestCommentLines and ignore the comments below. At least we should show an error if our tags appear there.